<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HalamanUtama</class>
 <widget class="QWidget" name="HalamanUtama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget #widget {
	background-image: url(:/gambar/Group 2.png);
}</string>
   </property>
   <widget class="QLineEdit" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushLogin">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>440</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
